frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -36,17 +36,23 @@ export interface OperatorValueSelectProps {
     operatorSelectProps?: Partial<Omit<LemonSelectProps<any>, 'onChange'>>
     eventNames?: string[]
     propertyDefinitions: PropertyDefinition[]
-    defaultOpen?: boolean
     addRelativeDateTimeOptions?: boolean
     groupTypeIndex?: GroupTypeIndex
     size?: 'xsmall' | 'small' | 'medium'
+    startVisible?: LemonDropdownProps['startVisible']
```

Comment:
we want to be able to start with the menu visible in stories so we can see when the allowlist kicks in